File: PY/905-sort-array-by-parity.py
# this uses a ternary to send each value to a placeholder list, then concatenates the lists
class Solution:
    def sortArrayByParity(self, A: List[int]) -> List[int]:
        odd = []
        even = []
        for i in A:
         odd.append(i) if i%2 != 0 else even.append(i)
        return even+odd

File: JS/905-sort-array-by-parity.js
/**
 * @param {number[]} A
 * @return {number[]}
 */
var sortArrayByParity = function(A) {
  odd = []
  even = []
  A.map(function(x,i) {
      return x % 2 != 0 ? odd.push(x) : even.push(x)
  })
  return(even.concat(odd))
};